ts/screens/profile/ShareDataScreen.tsx

```diff
@@ -27,11 +28,14 @@ type Props = ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps>;
 
 const ShareDataScreen = (props: Props): React.ReactElement => {
+  const store = useIOStore();
   const { present, bottomSheet } = useConfirmOptOutBottomSheet(() => {
```

Comment:
I understand that this PR isn't intended for this purpose, but I noticed that the `useConfirmOptOutBottomSheet` implementation relies on `useLegacyIOBottomSheetModal`, which is marked as deprecated. We now have `useIOBottomSheetModal` as the successor to `useLegacyIOBottomSheetModal`. While I'm not sure if updating `useConfirmOptOutBottomSheet` in this PR is appropriate, it certainly needs to be done to expedite the removal of the deprecated component.

Reply:
Yup! Good catch! But I prefer to leave this PR relative to the MP event only. We'll take care of a more deep refactoring of the page when we approach [this.]([IOPID-1506](https://pagopa.atlassian.net/browse/IOPID-1506)). What do you think?

Reply:
I totally agree with you!